src/NuGet.Core/NuGet.Protocol/Utility/CachingUtility.cs

```diff
@@ -50,11 +50,13 @@ public static Stream ReadCacheFile(TimeSpan maxAge, string cacheFile)
                         FileShare.Read | FileShare.Delete,
                         BufferSize);
 
-                    return stream;
+                    return (stream, false);
```

Comment:
Here `false` means cache is not expired.

Reply:
You might as well put a comment in the code here and line 62 with this info